storage/api/Storage.Samples.Tests/UploadObjectFromMemoryTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+using System.Linq;
+
+[Collection(nameof(StorageFixture))]
+public class UploadObjectFromMemoryTest
+{
+    private readonly StorageFixture _fixture;
+
+    public UploadObjectFromMemoryTest(StorageFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    [Fact]
+    public void UploadObjectFromMemory()
+    {
+        UploadObjectFromMemorySample uploadObjectFromMemorySample = new UploadObjectFromMemorySample();
+        ListFilesSample listFilesSample = new ListFilesSample();
+
+        uploadObjectFromMemorySample.UploadObjectFromMemory(_fixture.BucketNameGeneric, _fixture.FileName, "This is testing content");
```

Comment:
The fixture creates and object in the generic bucket with this file name. Which means that this test will always fail, or worst, it will override the existing file which is expected by some other test.

In general, the values that are defined on the fixture are to be used in tests that do not change them. Whatever your tests is going to change, the you need to create new and for your test only, and make sure that it is cleaned up afterwards. In this case you need to use a specific file name.